src/Component/Button.js

```diff
@@ -0,0 +1,12 @@
+function Button({type, onClick,className,children}) {
+    return (
+        <div><button
+            type={type}
+            onClick={onClick}
+            className={className}
+        >{children}
+        </button>
+        </div>
+    );
+}
```

Comment:
Dit is nu meer een doorgeefluik. Ten opzichte van het html button element zelf gebeurd er verder niets in dit component. Je had dus net zo goed direct kunnen gebruiken